src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -358,6 +358,24 @@ public IEnumerable<ActivityLink> Links
             }
         }
 
+        /// <summary>
+        /// Enumerate the tags attached to this Activity object.
+        /// </summary>
+        /// <returns><see cref="Enumerator{T}"/>.</returns>
```

Comment:
>Enumerator [](http://example.com/codeflow?start=32&length=10)

does this need to be qualified with `Activity.`

Reply:
I don't think so, but not 100% sure. As-is, it is resolving fine in VS. And if I make it `Activity.Enumerator{T}` I get a suggestion to simplify the name to `Enumerator{T}`.

Reply:
Interesting. Let's keep VS happy then and we can fix anything when porting the docs publicly to the docs repo. Good to know!